composer.json

```diff
@@ -19,21 +19,20 @@
             "Symfony\\WebpackEncoreBundle\\Tests\\": "tests/"
         }
     },
+    "minimum-stability": "dev",
```

Comment:
Did this stay on purpose ?

Reply:
Yes, it's a best practice to do this for libs.

Reply:
Thanks! I guess it's because it allows to install unstable deps easily when working on the bundle?